WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -1,39 +1,127 @@
 import Foundation
+import Reachability
+import RxCocoa
+import RxSwift
 
 let AccountSettingsServiceChangeSaveFailedNotification = "AccountSettingsServiceChangeSaveFailed"
 
-struct AccountSettingsService {
-    let remote: AccountSettingsRemote
+protocol AccountSettingsRemoteInterface {
+    var settings: Observable<AccountSettings> { get }
+    func updateSetting(change: AccountSettingsChange, success: () -> Void, failure: ErrorType -> Void)
+}
+
+extension AccountSettingsRemote: AccountSettingsRemoteInterface {}
+
+class AccountSettingsService {
+    struct Defaults {
```

Comment:
I was considering renaming `Defaults` to `Config`, since "Defaults" implies a value that can be overridden.